pgpkey/pgpkey_test.go

```diff
@@ -692,6 +696,17 @@ func TestCreateNewEncryptionSubkey(t *testing.T) {
 
 }
 
+func temporaryWorkAroundSetHashPreference(key *PgpKey) error {
```

Comment:
Cool, I've linked to this commit in our [trello card](https://trello.com/c/5IfrDbky/185-a-newly-generated-key-doesnt-have-any-cipher-hash-or-compression-preferences) (so let's not do any rebasing or it'll kill the hash)